tests/ignite/distributed/test_auto.py

```diff
@@ -164,7 +164,7 @@ def _test_auto_model_optimizer(ws, device):
     # Test auto_optim
     bnd = idist.backend()
     optimizer = optim.SGD(model.parameters(), lr=0.01)
-    optimizer = auto_optim(optimizer)
+    optimizer = auto_optim(optimizer, backward_passes_per_step=2)
```

Comment:
Let's create another specific test instead of overriding original one. Please, revert this change and add the following:
```python
if idist.has_hvd_support and bnd in ("horovod",):
    backward_passes_per_step = 2
    optimizer = optim.SGD(model.parameters(), lr=0.01)
    optimizer = auto_optim(optimizer, backward_passes_per_step=backward_passes_per_step)    
    assert isinstance(optimizer, optim.SGD) and hasattr(optimizer, "backward_passes_per_step")
    assert optimizer.backward_passes_per_step == backward_passes_per_step
```

Reply:
Sure.. I'll do that.